include/feedlistformaction.h

```diff
@@ -98,8 +98,6 @@ class FeedListFormAction : public ListFormAction {
 	FilterContainer& filters;
 
 	nonstd::optional<FeedSortStrategy> old_sort_strategy;
-
-	ListWidget feeds_list;
```

Comment:
You can probably remove `#include "listwidget.h"` as well, or at least move it to the .cpp file.

Reply:
Actually, removing it would be a waste of time: it doesn't impact compile times (that header is pulled via "listformaction.h" anyway) or anything else. So I'll just merge as-is. Sorry about the noise!

Reply:
I removed them anyway but added it as a small commit to PR #1080: https://github.com/newsboat/newsboat/pull/1080/commits/5d4d872b2ece52aff3031af921305d50d115cf4b